sgx-tools/go.sum

```diff
@@ -79,6 +79,16 @@ github.com/grpc-ecosystem/go-grpc-middleware v1.0.0/go.mod h1:FiyG127CGDf3tlThmg
 github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0/go.mod h1:8NvIoxWQoOIhqOTXgfV/d3M/q6VIi02HzZEHgUlZvzk=
 github.com/grpc-ecosystem/grpc-gateway v1.9.0/go.mod h1:vNeuVxBJEsws4ogUvrchl83t/GYV9WGTSLVdBhOQFDY=
 github.com/hashicorp/hcl v1.0.0/go.mod h1:E5yfLk+7swimpb2L/Alb/PJmXilQ/rhwaUYs4T20WEQ=
+github.com/hustliyilin/inclavare-containers/rune v0.0.0-20210207090805-681d56a22fa3 h1:Ga3ohjxqBlblONQ1baFWjZs9MnHCUWpxfQOnDRLcP3Y=
+github.com/hustliyilin/inclavare-containers/rune v0.0.0-20210207090805-681d56a22fa3/go.mod h1:zP5mWtsaryBVjmYViLmvFQutlnNLQaXIosgz0dhlu60=
+github.com/hustliyilin/inclavare-containers/rune v0.0.0-20210207100132-7ed4e6207c11 h1:EjyO7ldZPsRNhSDQMfvIvX+sa4UwUSZDK/G/5+F6cEE=
+github.com/hustliyilin/inclavare-containers/rune v0.0.0-20210207100132-7ed4e6207c11/go.mod h1:zP5mWtsaryBVjmYViLmvFQutlnNLQaXIosgz0dhlu60=
+github.com/hustliyilin/inclavare-containers/rune v0.0.0-20210207100835-c277e44d2129 h1:D88o2Qlqd5GyWym/20dkRJl8Z2EK+P6UIj3/c/mCdrI=
+github.com/hustliyilin/inclavare-containers/rune v0.0.0-20210207100835-c277e44d2129/go.mod h1:zP5mWtsaryBVjmYViLmvFQutlnNLQaXIosgz0dhlu60=
+github.com/hustliyilin/inclavare-containers/rune v0.0.0-20210207101336-67ea34af0e51 h1:ASRa1u41c3iUY5ywerM9TWjrvO2Qmgvn67DwbkVyRHA=
+github.com/hustliyilin/inclavare-containers/rune v0.0.0-20210207101336-67ea34af0e51/go.mod h1:zP5mWtsaryBVjmYViLmvFQutlnNLQaXIosgz0dhlu60=
+github.com/hustliyilin/inclavare-containers/rune v0.0.0-20210208031447-ea547001b6b5 h1:EGgI0FDfaZGaQsK60ojTgGPsnkzusWEgBmrd+R7BQ+8=
+github.com/hustliyilin/inclavare-containers/rune v0.0.0-20210208031447-ea547001b6b5/go.mod h1:zP5mWtsaryBVjmYViLmvFQutlnNLQaXIosgz0dhlu60=
```

Comment:
and these.

Reply:
> and these.

Done, thanks. The sgx-tools commit will place into another new pr after merging this pr to fix this problem.